frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -22,64 +22,84 @@ CELLxGENE is focused on supporting the global community attempting to create ref
 - drug screens
 - cell lines
 - organisms other than mouse or human
-- assays not on the [Census accepted assays list](https://github.com/chanzuckerberg/cellxgene-census/blob/main/docs/census_accepted_assays.csv)
+- assays not on the [CELLxGENE Census accepted assays list](https://github.com/chanzuckerberg/cellxgene-census/blob/main/docs/census_accepted_assays.csv)
+  - These additional assays are pending Census acceptance and will be accepted:
+    - Visium (non-HD)
+    - Slide-seq
+    - Expression measurements of multi-modal assays (e.g. 10x multiome, mCT-seq)
 
 ### Scale Constraints
 
 CELLxGENE Discover sets the maximum dataset size for submissions to 50GB. Additionally, CELLxGENE Explorer sets the maximum number of cells to 4.6 million for exploration of datasets.
 
 ### Formatting Requirements
 
-We need the following collection metadata (i.e. details associated with your publication or study)
+We need the following Collection metadata (i.e. details associated with your publication or study)
 
-- Collection information:
+- **Collection information**:
   - Title
   - Description
-  - Contact: name and email
-  - Publication/preprint DOI: can be added later
-  - URLs: any additional URLs for related data or resources, such as GEO or protocols.io - can be added later
-  - Consortia: optional, and can be added later. Can be one or more of those listed [here](https://github.com/chanzuckerberg/single-cell-data-portal/blob/main/backend/layers/common/validation.py#L12).
+  - Contact: a single name and email
+  - Publication/preprint DOI: *optional* can be added later
+  - URLs: *optional* can be added later. Any links to related data or resources, such as GEO or protocols.io
+  - Consortia: *optional* can be added later. Can be one or more of those listed [here](https://github.com/chanzuckerberg/single-cell-data-portal/blob/main/backend/layers/common/validation.py#L12)
 
-Each dataset needs the following information added to a single h5ad (AnnData 0.10) format file:
+The full schema is documented [here](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/5.2.0/schema.md) but is summarized below. Each dataset needs the following information added to a single h5ad (AnnData 0.10) format file:
```

Comment:
you can link out to https://chanzuckerberg.github.io/single-cell-curation/latest-schema.html, which will redirect to the whatever the latest schema is!

Reply:
brilliant